src/Service/Factory/SymfonyContainerFactory.php

```diff
@@ -31,13 +32,13 @@
      */
     public function createService(ServiceLocatorInterface $serviceLocator)
     {
-        return $this($serviceLocator, SymfonyContainerFactory::class);
+        return $this($serviceLocator, SymfonyContainerFactory::class, $serviceLocator->get('config'));
     }
 
     /**
      * {@inheritdoc}
      */
-    public function __invoke(ContainerInterface $zfContainer, $requestedName, array $options = null)
+    public function __invoke(ContainerInterface $zfContainer, $requestedName, array $config = null)
```

Comment:
I prefer you do not rename that parameter as in latest version of the zend-servicemanager it's called $options (https://github.com/zendframework/zend-servicemanager/blob/master/src/FactoryInterface.php#L21)

```suggestion
    public function __invoke(ContainerInterface $zfContainer, $requestedName, array $options = null)
```